src/Nethermind/Nethermind.Cli/NodeManager.cs

```diff
@@ -91,14 +92,9 @@ public async Task<JsValue> PostJint(string method, params object[] parameters)
                     decimal totalMicroseconds = stopwatch.ElapsedTicks * (1_000_000m / Stopwatch.Frequency);
                     Colorful.Console.WriteLine($"Request complete in {totalMicroseconds}μs");
                     string? resultString = result?.ToString();
-                    if (resultString == "0x" || resultString == null)
-                    {
-                        returnValue = JsValue.Null;
-                    }
-                    else
-                    {
-                        returnValue = _jsonParser.Parse(resultString);    
-                    }
+                    returnValue = resultString == "0x" || resultString == null
+                        ? JsValue.Null
+                        : _jsonParser.Parse(resultString);
```

Comment:
This was a compiler suggestion, I am curious what the team's preferences are for ternaries vs if blocks?

Reply:
compiler sugests it to the extreme which often leads to an unreadable code
our rule is that all coders have freedom to express their code in the way that they believe is most readable

we still have some minor arguments about the codestyle but generally there are not that many
we decided not to comment on codestyle during Code Reviews as it leads to disagreements - instead we focus on detecting logical or architectural problems or lack of test coverage